extensions/amp-story/1.0/_locales/en.json

```diff
@@ -218,5 +218,9 @@
   "101": {
     "string": "Shop Now",
     "description": "Label for the shopping CTA button."
+  },
+  "102": {
+    "string": "en-EN",
+    "description": "ISO 639-1 language code for English."
```

Comment:
Not this way :) 
The localization service works by reading the `lang` attribute of the viewer.
In that way we know that somehow our code is getting that value somewhere.
Checking out the `LocalizationService` class, we can see that it happens [here](https://github.com/ampproject/amphtml/blob/main/src/service/localization/index.js#L85).